data.sql

```diff
@@ -3,3 +3,53 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALU
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Gabumon', '2018-11-15', 1, true, 8);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Pikachu', '2021-1-7', 2, false, 15.04);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Devimon', '2017-5-12', 5, true, 11);
+
+/*Insert additional data to animals. */
+INSERT INTO animals (
+	name,date_of_birth,escape_attempts,neutered,weight_kg
+)
+VALUES (
+    'Charmander', '2020-02-08', 0, false, 11 
+);
+
+INSERT INTO animals (
+	name,date_of_birth,escape_attempts,neutered,weight_kg
+)
+VALUES (
+    'Plantmon', '2021-11-15', 2, true, 5.7 
+);
+
+INSERT INTO animals (
+	name,date_of_birth,escape_attempts,neutered,weight_kg
+)
+VALUES (
+    'Squirtle', '1993-04-02', 3, false, 12.13 
+);
+
+INSERT INTO animals (
+	name,date_of_birth,escape_attempts,neutered,weight_kg
+)
+VALUES (
+    'Angemon', '2005-06-12', 1, true, 45 
+);
+
+INSERT INTO animals (
+	name,date_of_birth,escape_attempts,neutered,weight_kg
+)
+VALUES (
+    'Boarmon', '2005-06-07', 7, true, 20.4 
+);
+
+INSERT INTO animals (
+	name,date_of_birth,escape_attempts,neutered,weight_kg
+)
+VALUES (
+    'Blossom', '1998-10-13', 3, true, 17 
+);
+
+INSERT INTO animals (
+	name,date_of_birth,escape_attempts,neutered,weight_kg
+)
+VALUES (
+    'Ditto', '2022-05-14', 4, true, 22 
+);
```

Comment:
- [ ] Please make sure that the entered data is the same as the data given in the exercise. Some of the given `weight_kg` values should be minus (`-`). This is intended and not a mistake as they are going to be fixed later with queries.

Reply:
Thank you @ShadyShawkat. I didn't notice the values were negative.